src/browser_action.js

```diff
@@ -370,11 +372,17 @@ browseraction.showEventsFromFeed_ = function(events) {
       $('<div>')
           .addClass('date-header')
           .text(headerDate.format('dddd, MMMM D'))
-          .appendTo($('#calendar-events'));
+          .appendTo(calendarEventsDiv);
     }
-
-    browseraction.createEventDiv_(event).appendTo($('#calendar-events'));
+    browseraction.createEventDiv_(event).appendTo(calendarEventsDiv);
   }
+
+  // Add 100ms delay to work around Chrome extension popup layout bug.
+  // See https://github.com/manastungare/google-calendar-crx/issues/224
+  // and the workarounds discussed in https://crbug.com/428044.
+  setTimeout( function () {
+    $('#calendar-events').replaceWith(calendarEventsDiv);
+  }, 100 );
```

Comment:
Interesting! Is the 100ms a magic number, or just something that clears the event loop before adding more DOM? Would 1ms work just as well, or does it have to be >=100ms?

Curious if we should make it a constant; that way we can experiment with the least value possible that still makes this work.

Reply:
I started with a value of 0 and went up in 25ms increments until I could no longer reproduce the bug. I updated the pull-request to make the value a constant and to add a comment explaining how it was chosen.